src/routes/safe/components/TransactionsNew/TxsTable/ExpandedTx/OwnersColumn/index.jsx

```diff
@@ -0,0 +1,119 @@
+// @flow
+import React, { useState } from 'react'
+import { List } from 'immutable'
+import { withStyles } from '@material-ui/core/styles'
+import Tabs from '@material-ui/core/Tabs'
+import Tab from '@material-ui/core/Tab'
+import Col from '~/components/layout/Col'
+import Row from '~/components/layout/Row'
+import Hairline from '~/components/layout/Hairline'
+import { type Owner } from '~/routes/safe/store/models/owner'
+import { type Transaction } from '~/routes/safe/store/models/transaction'
+import { TX_TYPE_CONFIRMATION } from '~/logic/safe/transactions/send'
+import { EMPTY_DATA } from '~/logic/wallets/ethTransactions'
+import OwnersList from './List'
+import ButtonRow from './ButtonRow'
+import { styles } from './style'
+
+type Props = {
+  tx: Transaction,
+  owners: List<Owner>,
+  classes: Object,
+  granted: boolean,
+  threshold: number,
+  userAddress: string,
+  thresholdReached: boolean,
+  safeAddress: string,
+  onTxConfirm: Function,
+  onTxCancel: Function,
+  onTxExecute: Function,
+}
+
+const isCancellationTransaction = (tx: Transaction, safeAddress: string) => !tx.value && tx.data === EMPTY_DATA && tx.recipient === safeAddress
```

Comment:
@germartinez What do you mean by valid format? `max-len` ESLint rule?

Reply:
yes

Reply:
@germartinez It doesn't, of course. For some reason prettier would always format it this way. Will fix

Reply:
@germartinez our precommit hook formats it this way, I can fix it but everytime you change the file it will bring it back to this format. Probably our ESLint rules need to be changed a little bit. Let's address that problem later?